docs/source/getting-started/install.rst

```diff
@@ -120,6 +120,9 @@ Installation as an Application: Downloading Releases
 
     ScanCode app archives come with packaged with all required dependencies except
     for Python that has to be downloaded and installed separately.
+    On more recent versions of Ubuntu, you'll have to install Python 3.8 manually.
+    One possibility to do so is to use Deadsnakes' PPA:
```

Comment:
This would make the indentation valid and highlight the following line:

```suggestion
    One possibility to do so is to use Deadsnakes' PPA::
```